modules/jolt_physics/objects/jolt_soft_body_3d.cpp

```diff
@@ -202,6 +202,11 @@ bool JoltSoftBody3D::_ref_shared_data() {
 		vertex_attrib.mCompliance = vertex_attrib.mShearCompliance = inverse_stiffness;
 
 		settings.CreateConstraints(&vertex_attrib, 1, JPH::SoftBodySharedSettings::EBendType::None);
+		if (shrinking_factor < 1) {
```

Comment:
Why limit to the range to [0, 1]? You could also use it to achieve the opposite effect.